src/features/shields/display.ts

```diff
@@ -0,0 +1,189 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from '../../theme'
+import palette from '../../theme/palette'
+import Button, { Props as ButtonProps } from '../../components/buttonsIndicators/button'
+import { ComponentType } from 'react'
+
+/**
+ * Header
+ */
+export const MainToggleHeading = styled<{}, 'h1'>('h1')`
+  box-sizing: border-box;
+  font-size: 16px;
+  line-height: 20px;
+  font-weight: 500;
+  color: ${palette.grey600};
+  margin: 0;
+`
+
+export const MainToggleText = styled<{}, 'p'>('p')`
+  box-sizing: border-box;
+  color: ${palette.grey600};
+  font-size: 12px;
+  font-family: ${p => p.theme.fontFamily.body};
+  margin: 0;
+  line-height: 1.2;
+`
+
+interface ToggleStateTextProps {
+  status: 'enabled' | 'disabled'
+}
+
+export const ToggleStateText = styled<ToggleStateTextProps, 'span'>('span')`
+  box-sizing: border-box;
+  color: ${p => p.status === 'enabled' ? palette.orange400 : palette.grey300};
```

Comment:
The `Down` color is a little light here and isn't passing AA color contrast checker so we should update since it's an important word. Anything grey700+ passes: https://d.pr/i/n1Onha

`color: ${p => p.status === 'enabled' ? palette.orange400 : palette.grey700};`

Reply:
My B. This ship has sailed, just realized the perogative for this is different than what I'm calling out. Will file issue for only this.